### PR TITLE
batches: update changelog for patch release

### PR DESCRIPTION
**Pull request by BolajiOlajide**
## Test plan

Changelog update

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -26,6 +26,7 @@ All notable changes to Sourcegraph are documented in this file.
 ### Fixed
 
 - Support expiring OAuth tokens for GitLab which became the default in version 15.0
```
Comment (unknwon) [inline CHANGELOG.md:28]: Unrelated to this PR:

@ryanslade I think we are missing a link to the PR here.

**Review by unknwon — APPROVED** [review]
Thank you!

**Comment by unknwon** [comment]
Merging as the release captain.

File: CHANGELOG.md (hunk 1 above)
Comment (ryanslade) [reply CHANGELOG.md:28]: I'll update it now.

**Pull request merged**